src/filter.jl

```diff
@@ -89,3 +172,54 @@ function backwardfilter(k::SDEKernel, (c, ν, P)::NamedTuple{(:logscale, :μ, :
 
   return message, sol[end]
 end
+
+
+Φfunc(t,T,B) = exp(-(T-t)*B)
+function solP(t,T,B,PT,Σ)
+  Φ = Φfunc(t,T,B)
+  Φ*(Σ+PT)*Φ'-Σ
+end
+
+function solν(t,T,B,β,νT)
+  #TODO
+  # β time-dependent
+  νT*Φfunc(t,T,-B) + Φfunc(t,T,-B)*β*inv(B)*(Φfunc(t,T,B)-1)
```

Comment:
It would be good to double check this line.

Reply:
I have a time-inhomogeneous example in https://github.com/mschauer/Bridge.jl/blob/430e9d8a0a39a2cd00f13cd10bfd1f3df5f3d2c4/src/linpro.jl#L13 Essentially we could try to reproduce tests which are using this https://github.com/mschauer/Bridge.jl/search?q=Ptilde